pkg/services/ngalert/api/tooling/definitions/api.go

```diff
@@ -24,11 +24,6 @@
 // swagger:meta
 package definitions
 
-// swagger:model
-type ValidationError struct {
```

Comment:
Moved to `shared.go` with the other similar responses and added an example comment.